### PR TITLE
Run ark in wasm

### PR DESCRIPTION
**Pull request by dark64**
https://github.com/Zokrates/ZoKrates/issues/1043

### CODE REVIEW

File: zokrates_core/src/ir/serialize.rs [hunk 1]
```diff
@@ -65,6 +65,15 @@ impl ProgEnum {
             Err(String::from("Wrong magic number"))
         }
     }
+
+    pub fn curve(&self) -> &'static str {
+        match self {
+            ProgEnum::Bls12_381Program(_) => Bls12_377Field::name(),
```
Comment (dark64) [inline zokrates_core/src/ir/serialize.rs]: todo: this is wrong

File: zokrates_cli/src/ops/verify.rs [hunk 2]
```diff
@@ -41,23 +41,23 @@ pub fn subcommand() -> App<'static, 'static> {
         .takes_value(true)
         .required(false)
         .possible_values(constants::BACKENDS)
-        .default_value(constants::BELLMAN)
+        .default_value(zokrates_common::constants::BELLMAN)
```
Comment (Schaeff) [inline zokrates_cli/src/ops/verify.rs]: Would it be acceptable to import `zokrates_common::constants`? This diff would be smaller and the code would be less verbose.

File: zokrates_js/Cargo.toml [hunk 3]
```diff
@@ -13,8 +13,11 @@ serde = { version = "^1.0.59", features = ["derive"] }
...
+zokrates_core = { path = "../zokrates_core", features = ["wasm", "ark"], default-features = false }
 zokrates_common = { path = "../zokrates_common" }
-zokrates_field = { path = "../zokrates_field", features = ["bellman"] }
+zokrates_field = { path = "../zokrates_field" }
 zokrates_abi = { path = "../zokrates_abi" }
-console_error_panic_hook = "0.1.6"
\ No newline at end of file
+console_error_panic_hook = "0.1.6"
+
+# [package.metadata.wasm-pack.profile.release]
```
Comment (Schaeff) [inline zokrates_js/Cargo.toml]: Does this have to be included?

File: zokrates_book/src/toolbox/zokrates_js.md [hunk 4]
```diff
@@ -133,14 +152,22 @@ Parameters:
 
 Returns: `SetupKeypair`
 
-##### exportSolidityVerifier(verificationKey, abi)
-Generates a Solidity contract which contains the generated verification key and a public function to verify proofs of computation of the compiled program.
+##### universalSetup(size)
+Performs the universal phase of a trusted setup. Only available for `marlin` scheme.
```
Comment (Schaeff) [inline zokrates_book/src/toolbox/zokrates_js.md]:
```suggestion
Performs the universal phase of a trusted setup. Only available for the `marlin` scheme.
```

**Review by Schaeff — CHANGES_REQUESTED** [review]
LGTM except that small typo!

**Review by Schaeff — APPROVED** [review]

**Pull request merged**